### ISSUE TITLE
[Task]: Setup Production API URL

### ISSUE THREAD

**Issue by acouch** [HHS/simpler-grants-gov#906]
### Summary

The production API will need a URL and DNS routing (ie `api.simpler.grants.gov`)

This is currently a security ticket as the HTTPs redirect will need to be in place to pass tests.

### Steps

- [x] Create CSR and DNS entry form to HHS. Same as https://github.com/HHS/simpler-grants-gov/issues/472
- [ ] Add domain and ACM to terraform. Similar to https://github.com/HHS/simpler-grants-gov/pull/557/files 

### Acceptance criteria

- [ ] The production URL is setup

**Comment by acouch** [comment]
This has been identified as a "nice to have" for v0.

**Comment by acouch** [comment]
This is currently blocked by a request to MH.

**Comment by sumiat** [comment]
Related to #1128

**Comment by sarahknoppA6** [comment]
@acouch @EOKENAVA Can we please get an update if this is still blocked? @lucasmbrown-usds @kw-MH

**Comment by coilysiren** [comment]
I have the last step here. My understanding is that this task is blocked on needing a DNS change and production certificate from HHS. Aaron submitted the form requesting those resources last week, and we have not heard back yet.

**Comment by coilysiren** [comment]
We have just received the production certificate from HHS, and they have performed the DNS change. I'll start work on this task ASAP

**Comment by coilysiren** [comment]
This might take me a few days. I have very low capacity this week, and its been a few years since I've done this.

**Comment by coilysiren** [comment]
https://api.simpler.grants.gov/docs is now live!
